_docs/services/aws-elasticache.md

```diff
@@ -12,11 +12,11 @@ cloud.gov offers [aws-elasticache-redis](https://aws.amazon.com/elasticache/) Re
 
 ## Plans
 
-Service Name | Plan Name | Description |
------------- | --------- | ----------- | -----
-`aws-elasticache-redis` | `redis-dev` | Single EC node for non-prod use only |
-`aws-elasticache-redis` | `redis-3node` | 3 node EC, persistent storage, 512Mb memory limit |
-`aws-elasticache-redis` | `redis-5node` | 5 node EC, persistent storage, 512Mb memory limit |
+Service Name | Plan Name | Description | Num of nodes |
```

Comment:
Can it say "number" instead of "num" just to avoid any possibility of confusion?

Reply:
adding now to see how it looks